tribes-react.md

```diff
@@ -0,0 +1,28 @@
+# Tribes
+
+## Description
+
+The purpose of this team is to create a strategy game based on the logic of [Travian](http://travian.com/). The application user will be a happy owner of a beautiful kingdom after the registration. The user has gold and food resources to feed the army, and to build a Farm, an Academy or a Mine. All the generation are time based, creating troops, buildings or upgrading them takes time based on levels. Users can attack, and occupy each others kingdom. There is a leaderboard where the users can monitor their progress.
+
+![](./assets/tribes-react.png)
+
+## Technologies
+
+In this project we are using React for the frontend, and Node.JS for the backend. First the apprentices learn the basics of React-Redux, Saga Middleware for the frontend part, and Mongoose at the backend, to easily handeling MongoDB data. Later they start testing everything both for the backend and the frontend.
```

Comment:
... and Saga middleware on the frontend. On the backend they use MongoDB which is accessed through the Mongoose library.